### PR TITLE
fixed typos

### PR DESCRIPTION
**Pull request by puntje**

### CODE REVIEW

File: lesson_0_generalization.txt [hunk 1]
```diff
@@ -243,17 +243,17 @@ Diagramming Tools
...
 
-You do not lose your previos work with the version control.
-Manual Commits. Give you more control on what en when is saved. By commenting every logical 
+You do not lose your previous work with the version control.
+Manual Commits. Give you more control on what and when something is saved. By commenting every logical 
 commit, you create handy overview.
 
 08/07/2016 14:00
-Multi-File commits by git are seen as dependencies. One file version control is ok for any small tasks. But 
-big projects requere more comlicated workflow with possibility to breack it up in pieces definde a bunch
```
Comment (puntje) [inline lesson_0_generalization.txt]: not sure what you meant by "definde a bunch of the steps"

File: lesson_2_reflections.txt [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
-	or is it local debuging or logging file, which most common not usefult to commmit.
+	staging area contains files that shoud be added from working directory to the repository,
+	it allows you to skip files that are not useful for other developers and are used only by you,
+	or those are local debugging or logging files, which are commonly not useful to commmit.
 
 How can you use the staging area to make sure you have one commit per logical
 change?
 
 	By adding files to the "staging area" we can commit changes to the repository in portions
-	of logical changes, howewer much more changes to more files have been made.
```
Comment (puntje) [inline lesson_2_reflections.txt]: am confused by __"however"__. I suggest: _"once enough changes have been made to the required amount of files."_

**Comment by puntje** [comment]
Only those with write access to this repository can merge pull requests.

**Comment by RelaRef** [comment]
Conclusion, it is secure :)
We will start collaborate work on our project soon!
Thank you :+1: for fixing my typos.

**Pull request merged**